### PR TITLE
2 packages from ocaml-community/cppo at 1.8.0

### PR DESCRIPTION
**Pull request by fpottier**
This pull-request concerns:
- `cppo.1.8.0`: Code preprocessor like cpp for OCaml
- `cppo_ocamlbuild.1.8.0`: Plugin to use cppo with ocamlbuild

---
* Homepage: https://github.com/ocaml-community/cppo
* Source repo: git+https://github.com/ocaml-community/cppo.git
* Bug tracker: https://github.com/ocaml-community/cppo/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

File: packages/cppo/cppo.1.8.0/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+homepage: "https://github.com/ocaml-community/cppo"
+doc: "https://ocaml-community.github.io/cppo"
+bug-reports: "https://github.com/ocaml-community/cppo/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {>= "2.0"}
+  "base-unix"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (avsm) [inline packages/cppo/cppo.1.8.0/opam:29]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/cppo_ocamlbuild/cppo_ocamlbuild.1.8.0/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+doc: "https://ocaml-community.github.io/cppo"
+bug-reports: "https://github.com/ocaml-community/cppo/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "2.0"}
+  "ocamlbuild"
+  "ocamlfind"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (avsm) [inline packages/cppo_ocamlbuild/cppo_ocamlbuild.1.8.0/opam:25]:
```suggestion
  ["dune" "subst"] {dev}
```

**Review by avsm — APPROVED** [review]
Just the subst -> dev change; none of the other revdep failures seem new from the previous release

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Comment by fpottier** [comment]
Thanks! I will make an announcement. I am not sure why the `opam` files contained the word `pinned`; they were created by `dune` and `opam publish`, and I believe that I did check that the package `cppo` was *not* pinned (on my local machine) before publishing. Oh, well.

**Pull request merged**